src/popover/popover.svelte

```diff
@@ -0,0 +1,18 @@
+<script>
+  import classes from '../_utils/classes.js';
+
+  let _class = null;
+  export { _class as class };
+  export let popoverClass = null;
+
+  export let position;
```

Comment:
Why not set default to bottom? It is what most people will probably use

Reply:
Well, in innopoints we only used the top popovers. Setting defaults makes sense when it's reasonable, in other words, when one option is clearly more attractive than the others, which isn't the case here. Best to make the devs be explicit about their intentions with this component

Reply:
In that case, specifying the position directly without having to import the enum will be much easier (`position="top"` instead of `position={PopoverPositions.TOP}`) since nothing forces users to use the enum anyways

Reply:
> In that case

In which case?

Yeah, I agree that specifying positions as strings would be easier, but I'd say that this is discouraged since strings may have typos and with enums we are free to change the internal values without breaking anyone's code

Reply:
> In which case?

The case of not having defaults.
Can you think of any good reason to change these values? I see it as just overhead for the user that they will have to import two distinct objects from different locations (the component and its enum)

Reply:
Changing values is not the reason. Error protection is. I'd say an enum is always better in such cases, even with another import